myriadeploy/deployment.cfg.local

```diff
@@ -16,3 +14,8 @@ rest_port = 8753
 [workers]
 1 = localhost:9001
 2 = localhost:9002
+
+[runtime]
+# Uncomment to set the maximum heap size of the worker processes
+#max_heap_size = -Xmx2g
```

Comment:
I guess we need to retain backward compatibility, but why do we need to use exactly the format of the JVM command-line option for heap size? It doesn't make any sense to someone unfamiliar with the JVM command line. How about a name like `max_heap_size_gb` so the value could be a simple number?